db/schema.rb

```diff
@@ -1287,6 +1287,17 @@
     t.datetime "rated_at", null: false
   end
 
+  create_table "scorecard_degree_programs", force: :cascade do |t|
+    t.integer "unitid"
```

Comment:
Do we need to set a primary key here?

Reply:
In this case, I don't believe we would since this field is not required and will end up with many records with the same `unitid`.